vmdb/db/migrate/20150401223517_container_nodes_hardware.rb

```diff
@@ -0,0 +1,16 @@
+class ContainerNodesHardware < ActiveRecord::Migration
```

Comment:
Ugh sorry...the name of this migration is extremely misleading.  I saw it and was thinking there was a **model** called `ContainerNodesHardware`  Please rename the migration to describe the change being made.